src/snowflake/snowpark/modin/plugin/docstrings/dataframe.py

```diff
@@ -922,6 +922,24 @@ def assign():
         2  22.0       12
         """
 
+    def align():
```

Comment:
Could you please follow the pattern of the other APIs in the file where we follow the pandas docs more closely (including description of parameters, examples, etc)

Reply:
Yes, added!